package.json

```diff
@@ -23,6 +20,9 @@
     "puppeteer": "0.12.0"
   },
   "devDependencies": {
-    "ava": "^0.24.0"
+    "ava": "^0.24.0",
```

Comment:
Think we don't need `ava` any more

Reply:
Still needs some cleaning I guess. It will be deleted.